app/views/layouts/application.html.erb

```diff
@@ -1,117 +1,110 @@
-<!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="utf-8" />
-  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-
-  <title><%= content_for?(:title) ? yield(:title) : "foundation-rails" %></title>
-
-  <%= stylesheet_link_tag    "application" %>
-  <%= javascript_include_tag "vendor/modernizr" %>
-  <%= javascript_include_tag "application", 'data-turbolinks-track' => true %>
-  <%= csrf_meta_tags %>
-  <link href="http://cdnjs.cloudflare.com/ajax/libs/foundicons/3.0.0/foundation-icons.css" rel="stylesheet">
-
-</head>
-
-<body>
-
-  <nav class="top-bar" data-topbar role="navigation">
-    <ul class="title-area">
-      <li class="name">
-        <h1><a href="<%= venues_path %> ">Jumper</a></h1>
-      </li>
-    </ul>
-
-    <section class="top-bar-section">
-      <!-- Right Nav Section -->
-      <ul class="right">
-        <li class="has-form">
-          <div class="row collapse">
-            <%= form_for venues_path, method: :get do %>
-              <div class="large-8 small-9 columns">
-                <%= text_field_tag :query, params[:query], attr: "Find Stuff" %>
-              </div>
-              <div class="large-4 small-3 columns">
-                <%= submit_tag "Search", name: nil %>
-              </div>
-            <% end %>
-          </div>
-        </li>
-        <li class="has-dropdown">
-          <a href="#">Filter by Boston Neighborhood</a>
-          <ul class="dropdown">
-            <li><a href="#">Back Bay</a></li>
-            <li><a href="#">Central Square</a></li>
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+    <title><%= content_for?(:title) ? yield(:title) : "foundation-rails" %></title>
+
+    <%= stylesheet_link_tag    "application" %>
+    <%= javascript_include_tag "vendor/modernizr" %>
+    <%= javascript_include_tag "application", 'data-turbolinks-track' => true %>
+    <%= csrf_meta_tags %>
+    <link href="http://cdnjs.cloudflare.com/ajax/libs/foundicons/3.0.0/foundation-icons.css" rel="stylesheet">
+    <link href='https://fonts.googleapis.com/css?family=Righteous' rel='stylesheet' type='text/css'>
+    <link href='https://fonts.googleapis.com/css?family=Open+Sans' rel='stylesheet' type='text/css'>
+    <link href='https://fonts.googleapis.com/css?family=Raleway' rel='stylesheet' type='text/css'>
```

Comment:
add the fonts in a sass file, probably call it `fonts.scss` and do something like (this) [http://www.rubyonrailsdeveloper.me/blog/import-google-fonts-in-rails]